app/assets/stylesheets/components/_footer.sass

```diff
@@ -0,0 +1,11 @@
+$Footer-white: Theme-color(background, footer)
+$Footer-blue: Theme-color(color, footer)
+$Footer-height: $Theme-height-footer
+
+.Footer
+  background: $Footer-white
```

Comment:
Don't use the [shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/background) unless it's actually intedend. Use the actual singular property `background-color`. This way you're not overwriting other `background-...` properties.